src/feature/dirauth/bwauth.c

```diff
@@ -205,7 +206,8 @@ dirserv_get_credible_bandwidth_kb(const routerinfo_t *ri)
 int
 dirserv_read_measured_bandwidths(const char *from_file,
```

Comment:
This function keeps on getting bigger. If we just add code, it will get harder and harder to change.

Let's think about splitting up what this function has to do into separate functions:
* read a line from the file
* add the line to the hash
* process the line
  * check the timestamp
    * add it to the headers
    * stop accepting timestamps
  * check for a k=v header, the terminator, or a relay line
    * add headers to the headers
    * at the terminator or first relay line, stop accepting k=v headers
    * add relay lines to the relays

We can make these changes *after* this branch is merged.